packages/core/contracts/financial-templates/common/FeePayer.sol

```diff
@@ -162,6 +162,7 @@ abstract contract FeePayer is AdministrateeInterface, Testable, Lockable {
      * @notice Removes excess collateral balance not counted in the PfC by distributing it out pro-rata to all sponsors.
      * @dev Multiplying the `cumulativeFeeMultiplier` by the ratio of non-PfC-collateral :: PfC-collateral effectively
      * pays all sponsors a pro-rata portion of the excess collateral.
+     * @dev This will revert if PfC is 0.
```

Comment:
i did not see the code that does this revert

Reply:
Its subtle but this [line](https://github.com/UMAprotocol/protocol/blob/master/packages/core/contracts/financial-templates/common/FeePayer.sol#L207) can revert because of a divide by 0 where PfC = 0 but balance > 0